pkg/analyzer/InsertReportManager.go

```diff
@@ -39,6 +39,8 @@ type RunResult struct {
   BuildC2 int `db:"build_c2"`
   BuildC3 int `db:"build_c3"`
 
+  TriggeredBy string `db:"triggeredBy"`
```

Comment:
We don't use clickhouse lib v1, and we don't use SQLX library. `db` annotation is not required (for "build" also should be removed)